example/public/script.js

```diff
@@ -4,6 +4,17 @@
 var incDom = document.querySelector('.increment');
 var decDom = document.querySelector('.decrement');
 var count = document.querySelector('.count');
+
+// Increment the counter when there's a click event on the + button
+incDom.addEventListener('click', function() {
```

Comment:
no harm in borrowing the code from: https://github.com/dwyl/learn-elm-architecture-in-javascript for this.  so the tutorials have continuity. 😉